checkov/terraform/checks/resource/digitalocean/SpacesBucketPublicRead.py

```diff
@@ -0,0 +1,23 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_negative_value_check import BaseResourceNegativeValueCheck
+from typing import List, Any
+
+
+class SpaceBucketPublicRead(BaseResourceNegativeValueCheck):
+    def __init__(self):
+        name = "Ensure the Spaces bucket is private"
+        id = "CKV_DIO_3"
+        supported_resources = ['digitalocean_spaces_bucket']
+        categories = [CheckCategories.BACKUP_AND_RECOVERY]
```

Comment:
```suggestion
        categories = [CheckCategories.NETWORKING]
```
probably fits better?

Reply:
I dunno. S3S3PublicACLRead uses GENERAL_SECURITY, maybe thats better?

Reply:
yeah, sounds also good 👍